Tools/ros2/ardupilot_msgs/msg/MotorControl.msg

```diff
@@ -0,0 +1,2 @@
+builtin_interfaces/Time timestamp # ms
```

Comment:
Can you add documentation what this message is for. Something like

"Control the direct motor outputs of the vehicle"

Reply:
Done!
Also I have modified the actuator array length to 32 in sync with AP_MOTORS_MAX_NUM_MOTORS.

Reply:
Nice, maybe later we can figure out how to use the define in the IDL and .msg file without duplicating it. Good enough for now.

Reply:
Alright!